Documentation/ApiOverview/FormEngine/Rendering/_ImportDataController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Controller\Ajax;
+
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+
+final class ImportDataController
+{
+    /**
+     * @param ServerRequestInterface $request
+     * @param ResponseInterface $response
+     * @return ResponseInterface
+     */
+    public function importDataAction(ServerRequestInterface $request, ResponseInterface $response): ResponseInterface
+    {
+        $queryParameters = $request->getParsedBody();
+        $id = (int)$queryParameters['id'];
```

Comment:
```suggestion
        $id = (int)($queryParameters['id'] ?? 0);
```

Maybe we may run into a warning when not having a fallback.

Reply:
Ah sorry, found another one without response.